README.md

```diff
@@ -54,6 +55,18 @@ const someModule = require('$utils/someModule')
 import someModule from '$utils/someModule'
 ```
 
+An *alternative* way is to call the betterModuleAlias function, and pass the aliases as the second argument.
+```js
+require('better-module-alias')(__dirname, {
+  $tests: './tests',
+  $utils: './utils',
+})
+```
+
+import betterModuleAlias from 'better-module-alias'
+betterModuleAlias(__dirname)
+```
```

Comment:
We should remove the 3 ticks on line `64` or add another 3 ticks and put "ESModules" before the second one.